src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -239,69 +240,75 @@ const ContentListTable = () => {
     meta: { count = 0 },
   } = data;
 
+  const repositoryUUID: string = '';
+
   const rowActions = useCallback(
-    (rowData: ContentItem): IAction[] =>
-      isRedHatRepository
-        ? features?.snapshots?.accessible
-          ? [
+    (rowData: ContentItem): IAction[] => {
+      const triggerSnapshotAction = {
+        isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
+        title: 'Trigger Snapshot',
+        onClick: async () => {
+          try {
+            await axios.post(`/api/content_sources/v1.0/repositories/${repositoryUUID}/snapshot/`);
```

Comment:
parameters look good, now you'll want to replace repositoryUUID with the rowData?.uuid  and i mentioned previously, instead of calling the API here, call the api function im ContentApi.ts   and pass the repository uuid  (using rowData?.uuid) to the api call function